src/librustc_monomorphize/partitioning.rs

```diff
@@ -922,7 +925,7 @@ fn collect_and_partition_mono_items<'a, 'tcx>(
 
     tcx.sess.abort_if_errors();
 
-    crate::monomorphize::assert_symbols_are_distinct(tcx, items.iter());
+    crate::assert_symbols_are_distinct(tcx, items.iter());
```

Comment:
Wait, is this the only call to this function? Can't it be moved into this module then? (And not made public?)

Reply:
To a large extent, I mostly just moved files. I didn't really do any deeper cleanup.

I'll move the function :+1: